src/routes/usersRoutes.ts

```diff
@@ -0,0 +1,15 @@
+import { Router, json } from 'express';
+import { userSchema } from '../validate/validate';
+import { createValidator } from 'express-joi-validation';
+import { createUser, getAllUsers, getUserById, updateUser, deleteUser, handleOtherPaths } from '../controllers';
+
+export const usersRoutes = Router();
+const validator = createValidator();
+
+usersRoutes
+  .get('/', getAllUsers)
+  .get('/:userId', getUserById)
+  .post('/', json(), validator.body(userSchema), createUser)
+  .put('/', json(), validator.body(userSchema), updateUser)
```

Comment:
```suggestion
  .post('/', validator.body(userSchema), createUser)
  .put('/', validator.body(userSchema), updateUser)
```
You could add one body parser at the top-level of your app (app.ts).
```
app.use(json());
```

Reply:
done